lansa-vmss-windows-autoscale-sql-database/azuredeploy.json

```diff
@@ -0,0 +1,829 @@
+                                 "properties": {
+                                    "subnet": {
+                                       "id": "[concat('/subscriptions/', subscription().subscriptionId,'/resourceGroups/', resourceGroup().name, '/providers/Microsoft.Network/virtualNetworks/', variables('virtualNetworkName'), '/subnets/', variables('subnetName'))]"
+                                    },
+                                    "loadBalancerBackendAddressPools": [
+                                       {
+                                          "id": "[concat('/subscriptions/', subscription().subscriptionId,'/resourceGroups/', resourceGroup().name, '/providers/Microsoft.Network/loadBalancers/', variables('dbloadBalancerName'), '/backendAddressPools/', variables('dbbePoolName'))]"
+                                       }
+                                    ]
+                                 }
+                              }
+                           ]
+                        }
+                     }
+                  ]
+               },
+               "extensionProfile": {
+                  "extensions": [
+                     {
+                        "name": "FirstInstall",
+                        "properties": {
+                           "publisher": "Microsoft.Compute",
+                           "type": "CustomScriptExtension",
+                           "typeHandlerVersion": "1.8",
+                           "autoUpgradeMinorVersion": "true",
+                           "forceUpdateTag": "[concat( parameters('installMSI'),parameters('updateMSI'),parameters('triggerWebConfig'),parameters('UninstallMSI'),parameters('fixLicense'))]",
+                           "settings": {
+                              "fileUris": [ "[variables('gitRefreshUri')]" ],
+                              "commandToExecute": "[concat('powershell -NoProfile -ExecutionPolicy unrestricted -command \"& {pushd;./',variables('gitRefreshName'), ' ', parameters('gitBranch'), ';popd;', 'pushd;c:\\lansa\\scripts\\azure-custom-script.ps1 ', '-server_name \"', reference(variables('sqlserverName')).fullyQualifiedDomainName, '\"', ' -DBUT \"SQLAZURE\"',  ' -dbname \"', parameters('databaseName'), '\"', ' -dbuser \"', parameters('databaseLogin'), '\"', ' -dbpassword \"', parameters('databaseLoginPassword'), '\"', ' -webuser \"', parameters('webUsername'), '\"', ' -webpassword \"', parameters('webPassword'), '\"', ' -MSIuri \"', parameters('msiUri'), '\"',  ' -maxconnections \"', parameters('webServerMaxConnect'), '\"', ' -trace \"', parameters('trace'), '\"', ' -installMSI \"', parameters('installMSI'), '\"', ' -updateMSI \"', parameters('updateMSI'), '\"',' -triggerWebConfig \"', parameters('triggerWebConfig'), '\"',' -UninstallMSI \"', parameters('UninstallMSI'), '\"', ' -fixLicense \"', parameters('fixLicense'), '\"', ';if ($LASTEXITCODE -ne 0) {Write-Error (\"MSI Install failed\");exit $LASTEXITCODE}; exit 0;popd;}\"')]"
```

Comment:
Ok. I applied `protectedSettings` and the deployment failed because the file in `fileUris` was not downloaded:
```
[1048+00000001] [11/09/2016 06:10:18.02] [INFO] Downloading files specified in configuration...
[1048+00000001] [11/09/2016 06:10:18.02] [INFO] No public configuration was provided, no files will be downloaded. Asynchronously executing command: 'SecureCommand_0'

```
Any ideas how to address this?